### PR TITLE
readme_table_content

### PR DESCRIPTION
**Pull request by memonsalvo**

### CODE REVIEW

**Review by esgaelramos — CHANGES_REQUESTED** [review]
Are you sure that is visible into the README?
please:
+ change the description of image for any like: 'Flow of Website...'
+ try to compress the gif, 20M its a lot
+ check if the urls of contents are correct (in work and syntax)

thanksss:)

**Comment by memonsalvo** [comment]
correcciones realizadas

**Comment by esgaelramos** [comment]
are you sure?

**Comment by memonsalvo** [comment]
Confirmed, the gif was compressed by 6% and the links were verified.

File: .github/fordevs/imgs/Flow_of_Website.gif
Comment (CamilaPua) [inline .github/fordevs/imgs/Flow_of_Website.gif:1]: i get a "Unable to render rich display" will be that a problem after the merge?

Comment (memonsalvo) [reply .github/fordevs/imgs/Flow_of_Website.gif:1]: I don't really know, woman, I owe it to you

**Comment by esgaelramos** [comment]
Re-Write please, in a branch

**Pull request closed without merging**